modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -1757,5 +1768,46 @@ def test_unsupported_columns(self, data, is_good):
             assert not obj and bad_cols == ["col"]
 
 
+class TestConstructor:
+    @pytest.mark.parametrize(
+        "index",
```

Comment:
index at the OmniSci backend is represented by hidden frame's columns, so we need to test shape detection with different kinds of indices to check that it's considering only data cols